LINK/usr/bin/miq_postgres_upgrade.sh

```diff
@@ -99,9 +104,41 @@ mount -a
 # init the new cluster
 su - postgres -c "initdb -D ${NEW_PGSQL_DIR}/data-new"
 
+sed -i.bak -e "s/^#*shared_preload_libraries.*/shared_preload_libraries = 'pglogical'/" ${NEW_PGSQL_DIR}/data-new/postgresql.conf
```

Comment:
That isn't really a problem either because we copy the existing config files over after the upgrade finishes, so these changes end up lost.